examples/run/CMakeLists.txt

```diff
@@ -27,3 +27,5 @@ if (OpenMP_CXX_FOUND)
     add_subdirectory(openmp)
     message(STATUS "OpenMP found! Include /examples/openmp subdirectory")
 endif()
+
+add_subdirectory(tbb)
```

Comment:
Are we not hiding these examples behind a configurable `TRACCC_BUILD_TBB` flag?

Reply:
Just read the description, still think this should be configurable.

Reply:
Moved the example under `cpu/`. Just to show that TBB should always be available going forward. :smile: